models/eprop_iaf_bsshslm_2020.h

```diff
@@ -88,49 +93,47 @@ able to spike for an absolute refractory period :math:`t_\text{ref}`.
 An additional state variable and the corresponding differential equation
 represents a piecewise constant external current.
 
-Furthermore, the pseudo-derivative of the membrane voltage needed for e-prop
-plasticity is calculated:
-
-.. math::
-    \psi_j^t = \frac{\gamma}{v_\text{th}} \text{max}
-               \left(0, 1-\left| \frac{v_j^t-v_\mathrm{th}}{v_\text{th}}\right| \right) \,.
-
-See the documentation on the ``iaf_psc_delta`` neuron model for more information
-on the integration of the subthreshold dynamics.
+See the documentation on the :doc:`iaf_psc_delta<../models/iaf_psc_delta/>` neuron model
+for more information on the integration of the subthreshold dynamics.
 
 The change of the synaptic weight is calculated from the gradient :math:`g` of
 the loss :math:`E` with respect to the synaptic weight :math:`W_{ji}`:
-:math:`\frac{\mathrm{d}{E}}{\mathrm{d}{W_{ij}}}=g`
+:math:`\frac{\text{d}{E}}{\text{d}{W_{ij}}}=g`
 which depends on the presynaptic
-spikes :math:`z_i^{t-1}`, the surrogate gradient / pseudo-derivative of the postsynaptic membrane
-voltage :math:`\psi_j^t` (which together form the eligibility trace
-:math:`e_{ji}^t`), and the learning signal :math:`L_j^t` emitted by the readout
-neurons.
+spikes :math:`z_i^{t-1}`, the surrogate gradient or pseudo-derivative
+of the spike state variable with respect to the postsynaptic membrane
+voltage :math:`\psi_j^t` (the product of which forms the eligibility 
+trace :math:`e_{ji}^t`), and the learning signal :math:`L_j^t` emitted
+by the readout neurons.
+
+See the documentation on the :doc:`eprop_archiving_node<../models/eprop_archiving_node/>` for details on the surrogate
+gradients functions.
 
 .. math::
-  \frac{\mathrm{d}E}{\mathrm{d}W_{ji}} = g &= \sum_t L_j^t \bar{e}_{ji}^t, \\
-   e_{ji}^t &= \psi^t_j \bar{z}_i^{t-1}\,, \\
+  \frac{ \text{d}E }{ \text{d}W_{ji} } &= g = \sum_t L_j^t \bar{e}_{ji}^t \,, \\
+   e_{ji}^t &= \psi^t_j \bar{z}_i^{t-1} \,, \\
 
 The eligibility trace and the presynaptic spike trains are low-pass filtered
 with some exponential kernels:
```

Comment:
```suggestion
The eligibility trace and the presynaptic spike trains are subjected to low-pass
filtering using exponential kernels:
```
I feel the word "some" is somewhat vague.

Reply:
I agree with your point that "some" is a bit vague here. However, I feel, the suggested solution expresses the very simple message in a bit too complicated way. What about simply and straightforward:

"The eligibility trace and the presynaptic spike trains are low-pass filtered with the following exponential kernels:"

Reply:
Yeah in my mind I removed "some" and felt something was missing. I ended up making the sentence more complex. Clearly the answer was to just replace "some" with something else!

Reply:
Fixed in https://github.com/JesusEV/nest-simulator/pull/16/commits/9163aedd43fd15810a7cf2151d23fa639b49dbe8